packages/clerk-js/src/ui/hooks/usePrefersReducedMotion.ts

```diff
@@ -13,9 +13,28 @@ export function usePrefersReducedMotion() {
       setPrefersReducedMotion(!event.matches);
     };
 
-    mediaQueryList.addEventListener('change', listener);
+    const addListenerCompat = (event: any, handler: any) => {
+      if ('addEventListener' in mediaQueryList) {
+        return mediaQueryList.addEventListener(event, handler);
+      } else {
+        // @ts-expect-error This is a fix for Safari iOS 13.4
+        // This version only supports the deprecated addListener method
+        // addListener accepts a single argument, which is the handler
+        return mediaQueryList.addListener(handler);
+      }
+    };
+
+    const removeListenerCompat = (event: any, handler: any) => {
+      if ('addEventListener' in mediaQueryList) {
+        return mediaQueryList.addEventListener(event, handler);
```

Comment:
⛔ i think we want to use `removeEventListener` in this and the above line.

Reply:
This is already fixed, please take another look :)